examples/USER/drude/ethylene_glycol/data.eg.lmp

```diff
@@ -0,0 +1,11259 @@
+created by fftool
```

Comment:
please follow the conventions for example file names in LAMMPS: for an example <name> the files are called, `in.<name>`, `data.<name>`, `log.<version>.<name>.g++.1` and `log.<version>.<name>.g++.4`. The reference data should be created with a binary compiled with GNU g++.